File: PHP PowerPoint Examples/LayoutSlidesApiDemo/getLayoutSlide.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\LayoutSlidesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new LayoutSlidesApi($configuration);

$name = "GetLayoutSlide.pptx";
$password = null;
$folder = "input";
$storage = null;
$slideIndex = 0;
$result = $apiInstance->getLayoutSlide($name,$slideIndex, $password, $folder, $storage);
print_r($result);

File: Python PowerPoint Examples/LayoutSlideApiDemo/getLayoutSlide.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.layout_slides_api.LayoutSlidesApi(configuration)

name = "GetLayoutSlide.pptx"
password = ""
storage = ""
folder = "input"
slideIndex = 0

result = api.get_layout_slide(name=name, slide_index=slideIndex,
                              password=password, storage=storage,
                              folder=folder)
